server/routes/interview.js

```diff
@@ -4,7 +4,11 @@ const { authMiddleware } = require("../middleware/authMiddleware");
 const InterviewController = require("../controllers/interviewController.js");
 
 // Use the authMiddleware to protect the /questions route
-router.get("/questions", authMiddleware, InterviewController.getQuestions);
+router.get(
```

Comment:
Create a new route for getting questions. The existing route is being used by admin dashboard to view all questions.